src/js/components/ServiceDetail.js

```diff
@@ -48,7 +50,13 @@ class ServiceDetail
   }
 
   renderTasksTabView() {
-    return (<span>Tasks Placeholder</span>);
+    let {service} = this.props;
```

Comment:
There is in general no need to cache those objects, as the performance benefits are in the mini micro scale, but that's a different story.